5-flexbox/assets/styles.css

```
@@ -0,0 +1,72 @@
+.content {
+    display: flex;
+    flex-direction: row;
+}
+
+img {
```

```suggestion
.hero-img {
```
Es recomendable crear clases que posteriormente puedan ser reusadas sin necesidad de cambiar la estructura del HTML.